man/zypper-docker-ps.1.md

```diff
@@ -0,0 +1,17 @@
+% ZYPPER-DOCKER(1) zypper-docker User manuals
+% SUSE LLC.
+% SEPTEMBER 2015
+# NAME
+zypper\-docker ps \- List all the running containers that are outdated.
+
+# SYNOPSIS
+**zypper-docker ps**
+
+# DESCRIPTION
+The **ps** command goes through all the running containers and lists which one
+of them are based on openSUSE/SLES and are outdated.
+
```

Comment:
I think we should extend that to make it clear the output is influenced by the data build over the time by patching and upgrading Docker images on the host running zypper-docker. I would not want people to think this command is going to provide feedback about all the possible suse containers running on the system.